src/main/src/main/java/org/geoserver/catalog/event/CatalogBeforeAddEvent.java

```diff
@@ -0,0 +1,13 @@
+/* (c) 2019 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.catalog.event;
+
+import org.geoserver.catalog.CatalogInfo;
+
+/** @author ImranR */
+public interface CatalogBeforeAddEvent extends CatalogEvent {
+    /** the object that was added. */
```

Comment:
This comment should not instead: "The object that is about to be added.", if I understood correctly the idea is to change the object before it gets stored by the catalog right?

Reply:
sorry comment is wrong, I will fix it, you got it right